roles/teuthology/tasks/main.yml

```diff
@@ -42,3 +42,11 @@
 - import_tasks: setup_log_access.yml
   tags:
     - logs
+
+- name: Enable and start beanstalkd
+  service:
+    name: beanstalkd
+    state: started
+    enabled: yes
+  tags:
+    - always
```

Comment:
Why the `always` tag here?

Reply:
I honestly don't know what for 'always' tag is used, it was just copy past from another service starting code in another role.
However I can say, that beanstalkd is a requirement for successful workers start or schedul request.

Reply:
It causes the task to be run regardless of `--tags` or `--skip-tags`; that won't be useful here, I don't think

Reply:
@zmc I've removed the tag.